libraries/chain/webassembly/privileged.cpp

```diff
@@ -145,6 +145,31 @@ namespace eosio { namespace chain { namespace webassembly {
       return context.control.set_proposed_producers( std::move(producers) );
    }
 
+   uint32_t interface::get_wasm_parameters_packed( span<char> packed_parameters ) const {
+      auto& gpo = context.control.get_global_properties();
+      auto& params = gpo.wasm_configuration;
+
+      auto s = fc::raw::pack_size( params );
+      if ( packed_parameters.size() == 0 )
+         return s;
+
+      if ( s <= packed_parameters.size() ) {
+         datastream<char*> ds( packed_parameters.data(), s );
+         fc::raw::pack(ds, params);
+         return s;
+      }
+      return 0;
```

Comment:
`return s;` to be consistent with the interface we have establish with `get_kv_parameters_packed`.